treebeard/templatetags/admin_tree.py

```diff
@@ -83,7 +83,7 @@ def get_result_and_row_class(cl, field_name, result):
             if isinstance(value, (datetime.date, datetime.time)):
                 row_classes.append('nowrap')
         else:
-            if isinstance(f.rel, models.ManyToOneRel):
+            if hasattr(f, 'rel') and isinstance(f.rel, models.ManyToOneRel):
```

Comment:
if we do it like this, that code will remain forever.
I would like to see a solution which can be safely be removed whenever we drop support for Django<2.0
Please use the version numbers provided by `django.__init__.py`